mocks/org_config.go

```diff
@@ -0,0 +1,22 @@
+package mocks
+
+import (
+	"context"
+
+	"github.com/influxdata/chronograf"
+)
+
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+type OrganizationConfigStore struct {
+	FindOrCreateF func(ctx context.Context, id string) (*chronograf.OrganizationConfig, error)
+	UpdateF       func(ctx context.Context, target *chronograf.OrganizationConfig) error
```

Comment:
*super nit* why call the argument here `target?

Reply:
this is what it was called in the `Config` code -- what do you propose? `upd`?